src/views/HelpAndKnowledge/SolutionsHelp/solutionsMap.tsx

```diff
@@ -82,7 +82,8 @@ export type ContactRoles =
   | 'Contracting Officer Representative, Division of Centralized Contracts and Services (DCCS)'
   | 'Quality Subject Matter Expert (QSME)'
   | 'Director, Division of Portfolio Management & Strategy'
-  | 'Director, Division of Budget and Administrative Services (DBAS)';
```

Comment:
Can you speak to why Product Manager is added here? I don't know why it's needed from the AC. Is it specifically to get the help center card working?

Reply:
I was trying to match Figma spec as closely as possible. Nicole Pham has the role "Product Manager" which does not otherwise exist, so I added it in order to assign her the appropriate role.

Reply:
Thanks Tom :)